src/Http/Http.Extensions/test/RequestDelegateFactoryTests.cs

```diff
@@ -2889,6 +2889,49 @@ public async Task RequestDelegateWithBindAndImplicitBodyRejectsNonJsonContent(bo
             }
         }
 
+        public static IEnumerable<object?[]> DateTimeDelegates
+        {
+            get
+            {
+                string dateTimeParsing(DateTime time) => $"Time: {time.ToString("O")}, Kind: {time.Kind}";
```

Comment:
I know it doesn't matter, but could we use the the culture specifier here regardless:

```suggestion
                string dateTimeParsing(DateTime time) => $"Time: {time.ToString("O", CultureInfo.InvariantCulture)}, Kind: {time.Kind}";
```